bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/component.tsx

```diff
@@ -167,6 +175,31 @@ const ChatMessageList: React.FC<ChatListProps> = ({
     }
   };
 
+  const renderUnreadNotification = () => {
+    if (totalUnread && userScrolledBack) {
+      return (
+        <UnreadButton
+          aria-hidden="true"
+          color="primary"
+          size="sm"
+          key="unread-messages"
+          label={intl.formatMessage(intlMessages.moreMessages)}
+          onClick={() => {
+            messagesEndRef.current?.scrollIntoView({ behavior: "smooth" })
+          }}
+        />
+      );
+    }
+    return null;
+  }
+
+  const handleScroll = (e: React.UIEvent<HTMLDivElement>) => {
```

Comment:
Please use [useCallback](https://react.dev/reference/react/useCallback#usecallback) to avoid re-declaration between  renders